cards/acute-limb-ischemia/card.md

```diff
@@ -34,16 +34,16 @@ categories:
 - **Neurological:** Check for sensory and motor loss
 - **Vascular:** Grade peripheral pulses
 
-- 0 = No Doppler signal
-- 1 = Reduced
-- 2 = Normal
-- 3 = Increased
-- 4 = Bounding
+    - 0 = No Doppler signal
```

Comment:
Is there a way that allows me to try to correct on Visual Code Editor after I do a pull request, and see that there's a build error? Or can I edit it here online somehow without breaking something more? Thx